CONTRIBUTING.md

```diff
@@ -83,19 +76,29 @@ if you choose to install `nextest`.
 
 ### Development
 
-After cloning the repository, run Ruff locally from the repository root with:
+After cloning the repository, Ruff can be run locally from the repository root on a file such as `add_rule.py` using:
 
 ```shell
-cargo run -p ruff -- check /path/to/file.py --no-cache
```

Comment:
It wasn't clear when this was run that the `/path/to/file.py` was an example.

This change updates this by clearing up the sentence, and also pointing to an example file that exists.